contracts/ERC20MultiDelegate.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
```

Comment:
An 'exchange' method that swaps tokens from one delegate to another would be nice to have, too, instead of having to withdraw and re-deposit.